stress_test.py

```diff
@@ -114,7 +114,7 @@ def pmem():
     '''
     Function to display memory statistics
     '''
-    tot, avail, percent, used, free = psutil.virtual_memory()
+    tot, avail, percent, used, free = psutil.virtual_memory()[:5]
```

Comment:
Thank you for your PR!

Return order of methods may be subject to a change in version number. Instead of grabbing the first 5 elements of the returned list, we can pick the specific ones we want.

```suggestion
    virtual_mem = psutil.virtual_memory()
    tot = virtual_mem.total
    avail = virtual_mem.available
    percent = virtual_mem.percent
    used = virtual_mem.used
    free = virtual_mem.free
```